src/components/pages/Friends/Friends.js

```diff
@@ -1,11 +1,72 @@
 import React from 'react';
+import smashRequests from '../../../helpers/data/smashRequests';
+import authRequests from '../../../helpers/data/authRequests';
+import FriendItem from '../../FriendItem/FriendItem';
 import './Friends.scss';
 
 class Friends extends React.Component {
+  state = {
+    users: [],
+    potentials: [],
+    confirmed: [],
+    pending: [],
+  }
+
+  componentDidMount() {
+    this.getAndSortUsers();
+  }
+
+  getAndSortUsers = () => {
+    const uid = authRequests.getCurrentUid();
+    smashRequests
+      .usersAndFriends(uid)
+      .then((results) => {
+        const users = results;
```

Comment:
you can delete this line since we don't need to keep track of the full users array